build.gradle

```diff
@@ -65,6 +65,26 @@ configurations {
     integrationTestRuntime.extendsFrom testRuntime
 }
 
+def getV8Dependency() {
+    def arch = System.getProperty('os.arch')
+    def dep = null
+    if (OperatingSystem.current().isWindows()) {
+        dep = 'win32_x86'
+        if (arch.equals('amd64')) {
+            dep += '_64'
+        }
+    } else if (OperatingSystem.current().isMacOsX() && (arch.equals('amd64') || arch.equals('x86_64'))) {
+        dep = 'macosx_x86_64'
+    } else if (OperatingSystem.current().isLinux() && arch.equals('amd64')) {
+        dep = 'linux_x86_64'
+    }
+    if (dep == null) {
+        logger.error("Could not find V8 runtime compatible to this system")
+        return null
```

Comment:
> What do you guys feel about just directly implementing citeproc natively in Java? I can take it up if you think that it will be worthwhile.

Hi all, CSL maintainer here. I have never worked on a citeproc implementation myself, but just a friendly warning that supporting the full CSL 1.0.1 specification is a significant undertaking. I'm cc'ing some developers who might give you some additional pointers/warnings.

(cc @michel-kraemer (citeproc-java), @fbennett (citeproc-js), @inukshuk (citeproc-ruby))